stdlib/generate/from_test.flux

```diff
@@ -21,7 +21,7 @@ testcase unionTwoStreamsWithEmptyGroupKeys {
     t2 =
         generate.from(
             count: 4,
-            fn: (n) => n * (-1),
+            fn: (n) => n * -1,
```

Comment:
This is sort of an interesting change.

Were the parens previously required for some reason or is this just the formatter stepping in to remove them now that the bind strength has been adjusted?

Reply:
Parenthesis is needed if a child node has lower precedence than its parent node. The same stands for the left child of a binary expression; while, for the right child, we need parenthesis if its precedence is lower or equal to its parent's.

As unary `-` has higher precedence now(earlier it was the same as binary `-`), formatter removed the parenthesis.

Reply:
Yep. Makes sense. It just looks so odd as it was originally and probably should have been an indication something was awry. Thanks :pray: